Sources/Action/UIKitExtensions/UIBarButtonItem+Action.swift

```diff
@@ -37,5 +38,28 @@ public extension Reactive where Base: UIBarButtonItem {
             }
         }
     }
+    public func bindToAction <Input,Output>(_ action:Action<Input,Output>?,_ inputTransform: @escaping (Base) -> (Input))   {
```

Comment:
Also, formatting wise, I don't think it would hurt to space things up a bit (space after colons and between parameters, for example.. )